chemprop/cli/utils/conf.py

```diff
@@ -1,7 +1,3 @@
 from datetime import datetime
-import os
-from pathlib import Path
 
-CKPT_DIR = Path(os.getenv("CHEMPROP_CKPT_DIR", "model_checkpoints"))
```

Comment:
I had added CKPT_DIR a while ago and my use case for it got removed. So this should be removed.